src/utils/wallet/providers/okx_wallet.ts

```diff
@@ -176,23 +175,46 @@ export class OKXWallet extends WalletProvider {
   };
 
   // Inscriptions are only available on OKX Wallet BTC mainnet (i.e okxWallet.bitcoin)
-  getInscriptions = async (): Promise<Inscription[]> => {
+  getInscriptions = async (): Promise<InscriptionIdentifier[]> => {
```

Comment:
Given that we replicate this function across two different wallets, can we have a util that does that?